graphstorm-processing/docker/0.4.0/requirements.txt

```diff
@@ -0,0 +1,942 @@
+attrs==23.2.0 ; python_full_version >= "3.9.12" and python_full_version < "3.10.0" \
```

Comment:
Why do we need the requirements.txt here?

Reply:
Previously we were using poetry to generate the requirements, now we pre-generate this, once per version and include it with the Dockerfiles.